lib/meandro/rules/unused_struct_field.ex

```diff
@@ -0,0 +1,149 @@
+defmodule Meandro.Rule.UnusedStructField do
+  @moduledoc """
+  Finds struct fields that are not used.
+  It has the following assumptions:
+    - As we are not tracking the value of each variable and a variable can be
+      reassigned we don't check the struct name when looking for access to an
```

Comment:
nit: a struct, not an struct
```suggestion
      reassigned we don't check the struct name when looking for access to a
```